docs/reference/setup/cluster_restart.asciidoc

```diff
@@ -30,55 +33,108 @@ PUT _cluster/settings
 // TEST[skip:indexes don't assign]
 --
 
-. *Perform a synced flush*
+. *Stop indexing and perform a synced flush.*
 +
 --
 
-Shard recovery will be much faster if you stop indexing and issue a
-<<indices-synced-flush, synced-flush>> request:
+Performing a <<indices-synced-flush, synced-flush>> speeds up shard
+recovery.
 
 [source,sh]
 --------------------------------------------------
 POST _flush/synced
 --------------------------------------------------
 // CONSOLE
 
-A synced flush request is a ``best effort'' operation. It will fail if there
-are any pending indexing operations, but it is safe to reissue the request
-multiple times if necessary.
+Synced flush operations are performed on a "best effort" basis. The request
+fails if there are any pending indexing operations, but it is safe to
+reissue the request multiple times until it succeeds.
 --
 
-. *Shutdown and upgrade all nodes*
+. *Shutdown all nodes.*
 +
 --
+* If you are running Elasticsearch with `systemd`:
++
+[source,sh]
+--------------------------------------------------
+sudo systemctl stop elasticsearch.service
+--------------------------------------------------
 
-Stop all Elasticsearch services on all nodes in the cluster. Each node can be
-upgraded following the same procedure described in <<upgrade-node>>.
+* If you are running Elasticsearch with SysV `init`:
++
+[source,sh]
+--------------------------------------------------
+sudo -i service elasticsearch stop
+--------------------------------------------------
+
+* If you are running Elasticsearch as a daemon:
++
+[source,sh]
+--------------------------------------------------
+kill `cat pid`
```

Comment:
I believe `kill $(cat pid)` is preferred lately. I think the backticks were ambiguous in some scripts so we've all been slowly shifting to `$(` and `)`.